students/anjehle/lesson02/fizzbuzz.py

```diff
@@ -0,0 +1,10 @@
+for i in range(1,101):
+	if i % 3 == 0:
+		if i % 5 == 0:
+			print("FizzBuzz")
+		else:
+			print("Fizz")
+	elif i % 5 == 0:
+		print("Buzz")
+	else:
+		print(i)
```

Comment:
the code looks good - but you want to make sure you're using exactly 4 spaces for indentation per python best practices.

your editor can be configured or you can use Pycharm which comes built in with the correct settings.